lang/javascript/javascript.py

```diff
@@ -1,11 +1,44 @@
-from talon import Module, Context, actions, settings
+from talon import Context, Module, actions, settings
 
 mod = Module()
 ctx = Context()
 ctx.matches = """
 tag: user.javascript
 """
 
+ctx.lists["user.code_common_function"] = {
+    "abs": "Math.abs",
+    "entries": "Object.entries",
+    "flatten": "flatten",
```

Comment:
Just wondering where this comes from?

Reply:
ah right; looks like this one should be `flat`?

Reply:
Would that be referring to Array#flat? I guess then it should be in the other list.

Reply:
ah wait nvm this is in common functions.  It's from lodash.  Shall we leave it out since it's not standard?

Reply:
Deleted, since it's not standard

Reply:
Is `flat` common enough to include?  I could go either way

Reply:
Btw tempted to make these into csvs so users can add their own common functions

Reply:
I've honestly never seen flat used.